plugin.js

```diff
@@ -440,6 +460,37 @@
 			return items.length;
 		};
 
+		/**
+		 * Suppresses change events until after delegateFunction has been executed.
+		 * 
+		 * Performing multiple successive updates to a PlaceholdersCollection object

```

Comment:
I don't really like that the collection is "aware" of UI problems. This is something that is better off being documented elsewhere, I'll add some explanation for dynamic placeholders in the project docs later on, I think it would fit there. Having a method that suppresses events for successive operations is generally useful, so the description should just explain what the method does.